connector/http-sink-connector/README.md

```diff
@@ -0,0 +1,157 @@
+# HTTP sink Connector
+
+In this example, you'll setup the followings:  
+* nodejs as an API server 
+* Confluent Platform with Connect and http sink connector.  
+
+You will install and manage the https sink connector and plugin jars through the declarative `Connector` CRD.  
+In this scenario, we deploy the a Confluent cluster without security.  
+
+## Set up Pre-requisites
+
+Set the tutorial directory for this tutorial under the directory you downloaded
+the tutorial files:
+
+```
+export TUTORIAL_HOME=<Tutorial directory>/connector/http-sink-connector
+```
+
+Create a namespace:  
+```
+kubectl create ns confluent
+```
+
+Deploy Confluent for Kubernetes (CFK): 
+
+```
+helm upgrade --install confluent-operator \
+  confluentinc/confluent-for-kubernetes \
+  --namespace confluent
+```
+
+## Deploy nodejs API server   
```

Comment:
maybe explain that the webserver is a custom built image and how it works. accepts simple PUT requests? and stores all messages on an endpoint